src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  return (
```

Comment:
```suggestion
export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
```

can be a bit simplify